interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: I am not entirely sure but I think that "this" is a part of react that can be used in Javascript. I think it may involve using functions. 
 
-  Researched answer:
+  Researched answer: "This" is a key word in javascript that refers to an object. It is used in a function to show where to get value or data from. 
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a javascript Library. You would use it to access a larger ammount of keyword and abilities that you wouldn't be able to do normally in JS.
 
-  Researched answer:
+  Researched answer: Created by FaceBook, React is a front-end JS libray that is free and open source. It is used to build user interfaces based on UI components.
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: Not entirely sure what a lifecycle method is, would have to review some notes and do some more research. 
 
-  Researched answer:
+  Researched answer: The only method required in a class component is the Render() Method.
 
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
-  Your answer:
+  Your answer: Not entirely sure but I would guess that JSX is the ability to do HTML work and styling inside of a JS file. The difference may be that you can only do HTML in JS but cannt do JS in HTML.
 
-  Researched answer:
+  Researched answer: JSX is a language syntax extension for JS. It has a similar appearence to HTML and provides aa way to structure component rendering. 
```

Comment:
Very important to React. Let's you do the logic like JS but display it as HTML on the screen.